elastic/kibana
# [Ingest Node Pipelines] New patterns component for Grok processor

## Summary

Adds a new component for managing patterns in the grok processor form. This will replace the combobox component that is there currently.

## How to test

1. Go to ingest node pipelines in management section
2. Create a new pipeline
3. Add a grok processor
4. Input some patterns
5. Reorder them
6. Save the processor
7. Open the processor and move the patterns, save and check state (try removing all patterns)
8. Save pipeline


## Screenshot

<img width="696" alt="Screenshot 2020-09-18 at 11 56 31" src="https://user-images.githubusercontent.com/8155004/93584922-50943900-f9a6-11ea-989e-d01d2475b687.png">

<img width="698" alt="Screenshot 2020-09-18 at 11 56 40" src="https://user-images.githubusercontent.com/8155004/93584925-51c56600-f9a6-11ea-86e3-f6a94e6bc602.png">

<img width="701" alt="Screenshot 2020-09-18 at 11 56 52" src="https://user-images.githubusercontent.com/8155004/93584930-525dfc80-f9a6-11ea-94ef-f7c42d41de0b.png">



### Checklist

Delete any items that are not applicable to this PR.


### For maintainers
